components/builder-depot/Cargo.toml

```diff
@@ -64,7 +64,7 @@ git = "https://github.com/habitat-sh/urlencoded.git"
 branch = "habitat"
 
 [dependencies.zmq]
-git = "https://github.com/erickt/rust-zmq.git"
+git = "https://github.com/habitat-sh/rust-zmq.git"
```

Comment:
We can also pin to a specific branch, tag, or revision, or maybe add a comment to explain why we are currently binding to a fork here and what the conditions are to go back to the standard crate.

http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-git-repositories